### DISCUSSION TITLE
On the Accuracy of Division Calculation

### DISCUSSION THREAD

**Discussion by huaizhe2012** [deepjavalibrary/djl#2093] [Q&A] [answered]
Hello, My data precision is eight decimal places, but after division, the precision becomes four. What can I do to keep its precision，thanks.
![0KH1R8OT_@ASD2MF DURI{I](https://user-images.githubusercontent.com/49636762/196912627-d9fed72c-f32f-4135-9dc8-2c5bea7920e0.png)

**Comment by frankfliu** [comment, marked as answer]
This is just display issue. DJL trim the trailing 0s when print the NDArray.

You can dump the NDArray using `NDArray.toNumber()` to verify the precision.

**Reply by huaizhe2012** [reply]
This is really a display issue，Thank you very much
